pkgs/development/libraries/libvori/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, fetchurl, cmake } :
+
+stdenv.mkDerivation rec {
+  pname = "libvori";
+  version = "210412";
+
+  nativeBuildInputs = [ cmake ];
+
+  # Original server is misconfigured and messes up the file compression.
```

Comment:
The original download location seems to work for me: `https://brehm-research.de/files/libvori-210412.tar.gz`

Reply:
Nice, so this has been fixed recently. Changed it.